mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -326,6 +348,46 @@ impl RingBufferSettings {
     }
 }
 
+#[derive(Deserialize)]
+#[serde(untagged)]
+enum NonZeroU64OrOther {
+    U64(u64),
+    STR(String),
+}
+
+fn deserialize_with_u64<'de, D>(deserializer: D) -> Result<NonZeroU64, D::Error>
```

Comment:
Should allow for conversion of string or int now, which will make env var or json easy to use